taskcluster/.shared.yml

```diff
@@ -137,14 +137,14 @@ system:
     namespace: 'project.deepspeech.gradle.7'
   pyenv:
     linux:
-      url: 'https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.pyenv.linux.9/artifacts/public/pyenv.tar.gz'
-      namespace: 'project.deepspeech.pyenv.linux.9'
+      url: 'https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.pyenv.linux.10/artifacts/public/pyenv.tar.gz'
+      namespace: 'project.deepspeech.pyenv.linux.10'
```

Comment:
It looks like I had some previous experiment that already made use of `10`, please use 11